task-01/task_01.md

```diff
@@ -195,4 +206,4 @@ Submit the following files packaged into a zip file named `task01.zip`:
 #### Presentation
 
 For each group, one student representative demonstrates their Hadoop setup in action for selected configuration settings. This includes (re)running the MapReduce program and showing and interpreting the effects of the configuration via the Hadoop and Grafana web UIs.  
-Each lab session, we rotate and document the students representing the groups so every student gets her/his turn in the course of the semester.
\ No newline at end of file
+Each lab session, we rotate and document the students representing the groups so every student gets her/his turn in the course of the semester.
```

Comment:
I am missing something here, what did you change?

Reply:
No change intended there, I don't know.